plugins/dynamic-frontend-plugin-sample/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@backstage/plugin-dynamic-frontend-plugin-sample",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "workspace:^",
+    "@backstage/core-plugin-api": "workspace:^",
+    "@backstage/theme": "workspace:^",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^",
+    "@backstage/core-app-api": "workspace:^",
+    "@backstage/dev-utils": "workspace:^",
+    "@backstage/test-utils": "workspace:^",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^14.0.0",
+    "msw": "^1.0.0"
+  },
+  "scalprum": {
+    "name": "backstage.dynamic-frontend-plugin-sample",
+    "exposedModules": {
+      "./SampleComponent": "./src/SampleComponent"
+    }
+  },
```

Comment:
The design in the new frontend DI system (and backend too) is to export the main feature as the default export of the package, with all other extensions being provided by that export. How do you feel about implementing that straight away here?

Reply:
Could be. I am not stuck on this configuration. You have to take into consideration the option of exposing multiple entry points of a single plugin.

But if I understand it correctly, we want a config like: 

```js
{
  name: "foo",
  exposedModules: {
    default: "path/to/source"
  }
}
```

Reply:
Tbh we don't really want any config ay all here. We're looking to not have multiple entry points, just a single one that can expose multiple extensions, some of which are disabled by default.

Reply:
> Tbh we don't really want any config at all here.
Yeah, I got that. I am thinking more in a way about what to "feed" to the plugin configuration. So to me, it seems like we want the `main` field from the package.json as the entry point or path to the exposed module (index.ts) and name it `default`.